portfolio/pom.xml

```diff
@@ -43,10 +43,10 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <!-- TODO: uncomment configuration block and provide project ID. -->
```

Comment:
nit: You can probably delete this comment now.

Reply:
Done